.github/workflows/tap-tests.yml

```diff
@@ -33,37 +36,64 @@ jobs:
           sudo -E apt-get install krb5-admin-server krb5-kdc krb5-user libkrb5-dev -y -qq
         shell: bash
 
-      - name: Build Modified Postgres
-        id: build-modified-postgres
+      - name: Build Modified Postgres using ${{env.ENGINE_BRANCH_OLD}}
```

Comment:
are we always creating both old and new installations?

Reply:
Yep, need both installations to verify if we correctly blocking cross-version dump/restore.